tests/test_get_followers_notworkingyet (2).py

```diff
@@ -1,7 +1,7 @@
 # test_get_following.py
 import unittest
 from unittest.mock import patch, MagicMock
-from researchtikpy.src.get_following import get_following
```

Comment:
This line was giving me import errors, so I had to change it. Same for the other test file.